### DISCUSSION TITLE
Where can i find the un-uglified version of xterm.js file?

### DISCUSSION THREAD

**Discussion by hannes7g** [sourcelair/xterm.js#4629] [Q&A]
I am not a web developer, i have to no idea what i should do to create xterm.js file as it is not present on this repo. unuglifying the xterm.js online did not bring much either. Because i see variable names like i,f,t,..... which are not intuitive. variable names should be able to tell what they are.
i'd really appreicate the support
thanks

**Comment by Tyriar** [comment]
If you want the plain source you can clone the repo and run yarn, that will install dependencies and build the project.

**Comment by hannes7g** [comment]
is there not any otherway?  installing yarn seems like an overkill
https://phoenixnap.com/kb/yarn-windows
also i dont have any project, i am just using notepad. i dont know what "that will install dependencies and build the project." means

**Reply by Tyriar** [reply]
If you just want to look at the source code it's available as TypeScript in the repo, to get JS you need to build it though as we only publish the minified code.

**Reply by hannes7g** [reply]
Thanks for the help Tyriar! I guess i have to use yarn at a later point. I dont know how to build it but i just have figure it out somehow by reading something. I also don't know TypeScript. Another language is too much pita for me at the moment.

**Reply by Tyriar** [reply]
You should just be able to install it with via yarnpkg.com and also install node.js
